default/scripting/common/enqueue.macros

```diff
@@ -1,3 +1,33 @@
+/* Allows the current building to be enqueued if the given prerequisite is built or enqueued.
+   Takes the prerequisite name as parameter; Usage:
+        [[LOCATION_ALLOW_ENQUEUE_IF_ENQUEUED(BLD_SHIPYARD_BASE)]]
+*/
+LOCATION_ALLOW_ENQUEUE_IF_ENQUEUED
```

Comment:
I think making this macro name slightly longer would make it much more clear-- 
```
LOCATION_ALLOW_ENQUEUE_IF_PREREQ_ENQUEUED
```

Reply:
sounds good to me